ds3_java_cli/src/main/java/com/spectralogic/ds3cli/command/CliCommand.java

```diff
@@ -29,8 +31,23 @@
 
 public abstract class CliCommand<T extends Result> implements Callable<T> {
 
-    private final Ds3Provider ds3Provider;
-    private final FileUtils fileUtils;
+    private Ds3Provider ds3Provider;
+    private FileUtils fileUtils;
+
+    protected final View<T> cliView = (View<T>) new DefaultView();
```

Comment:
Do you have to cast this with the `(View<T>)`?  So long as DefaultView is implemented to return a T that extends Result, you shouldn't have to cast it...